orm6/integrationtest/java/modules/orm-coordination-outbox-polling-elasticsearch/pom.xml

```diff
@@ -54,6 +54,12 @@
             <version>${version.net.bytebuddy}</version>
         </dependency>
 
+        <!-- Only needed here so we can require it in module info for Avro -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
```

Comment:
Same comment as elsewhere, shouldn't this be `log4j-api`, and can we reduce the scope?